test/CMakeLists.txt

```diff
@@ -47,7 +64,7 @@ if (WITH_TEST_GENERATORS)
   # and make an executable target for it named 'generator_foo'
   file(GLOB TESTS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/generator" "${CMAKE_CURRENT_SOURCE_DIR}/generator/*_jittest.cpp")
   foreach(test ${TESTS})
-    string(REPLACE "_jittest.cpp" "" name "generator_jit_${test}")
+    string(REPLACE "_jittest.cpp" "" name "generator_exec_jit_test_${test}")
```

Comment:
You would think that 256 byte (not character) path length (not including drive name and terminating NUL) would be a stupid limitation, but it is a carry over from the BIOS call that returns the length in an 8-bit register. Microsoft doesn't like to deprecate anything, ever, if it can possibly be avoided, so applications written 20 years ago can still work. Fixing it means touching code all over the build tools that assume a buffer of 260 bytes is enough. On the other hand, Apple have changed CPUs twice (Moto 68K -> Power -> Intel x86), totally rebased the OS and apps that were written for the previous OS update may stop working.